modules/products/bitbucket/variables.tf

```diff
@@ -178,8 +178,4 @@ variable "shared_home_snapshot_id" {
   description = "EBS Snapshot ID with shared home content."
   type        = string
   default     = null
-  validation {
```

Comment:
The validation was too restrictive? Why we are removing it?

Reply:
We already have the exact same validation at NFS module level, which I think is the better place to enforce the snapshot id pattern.